tests/conftest.py

```diff
@@ -13,219 +13,291 @@
 from trailblazer.mip import files as mip_dna_files_api
 
 pytest_plugins = [  # pylint: disable=invalid-name
-    'tests.apps.lims.conftest',
-    'tests.apps.loqus.conftest',
-    'tests.cli.conftest',
-    'tests.delivery.conftest',
-    'tests.delivery.conftest',
-    'tests.meta.conftest',
-    'tests.meta.orders.conftest',
-    'tests.meta.report.conftest',
-    'tests.meta.transfer.conftest',
-    'tests.meta.upload.conftest',
-    'tests.store.conftest',
-    'tests.apps.mutacc_auto.conftest'
+    "tests.apps.lims.conftest",
+    "tests.apps.loqus.conftest",
+    "tests.cli.conftest",
+    "tests.delivery.conftest",
+    "tests.delivery.conftest",
+    "tests.meta.conftest",
+    "tests.meta.orders.conftest",
+    "tests.meta.report.conftest",
+    "tests.meta.transfer.conftest",
+    "tests.meta.upload.conftest",
+    "tests.store.conftest",
+    "tests.apps.mutacc_auto.conftest",
 ]
 
 
+@pytest.fixture
```

Comment:
I suggest moving this to tests/apps/madeline/conftest.py unless there is a good reason to have it here?